modules/libarchive/3.7.5.bcr.1/overlay/libarchive/test/BUILD.bazel

```diff
@@ -0,0 +1,912 @@
+libarchive_test(name = "test_read_format_gtar_sparse_skip_entry", deps = [":lib"])
+libarchive_test(name = "test_read_format_iso_Z", deps = [":lib"])
+libarchive_test(name = "test_read_format_iso_multi_extent", deps = [":lib"])
+libarchive_test(name = "test_read_format_iso_xorriso", deps = [":lib"])
+libarchive_test(name = "test_read_format_isojoliet_bz2", deps = [":lib"])
+libarchive_test(name = "test_read_format_isojoliet_long", deps = [":lib"])
+libarchive_test(name = "test_read_format_isojoliet_rr", deps = [":lib"])
+libarchive_test(name = "test_read_format_isojoliet_versioned", deps = [":lib"])
+libarchive_test(name = "test_read_format_isorr_bz2", deps = [":lib"])
+libarchive_test(name = "test_read_format_isorr_ce", deps = [":lib"])
+libarchive_test(name = "test_read_format_isorr_new_bz2", deps = [":lib"])
+libarchive_test(name = "test_read_format_isorr_rr_moved", deps = [":lib"])
+libarchive_test(name = "test_read_format_isozisofs_bz2", deps = [":lib"])
+libarchive_test(name = "test_read_format_lha", deps = [":lib"])
+libarchive_test(name = "test_read_format_lha_bugfix_0", deps = [":lib"])
+libarchive_test(name = "test_read_format_lha_filename", deps = [":lib"])
+libarchive_test(
+    name = "test_read_format_lha_filename_utf16",
+    deps = [":lib"],
+    tests = [
+        "test_read_format_lha_filename_UTF16",
+    ],
+)
+libarchive_test(name = "test_read_format_mtree", deps = [":lib"])
+libarchive_test(name = "test_read_format_mtree_crash747", deps = [":lib"])
+libarchive_test(name = "test_read_format_pax_bz2", deps = [":lib"])
+libarchive_test(
+    name = "test_read_format_rar",
+    deps = [":lib"],
+    tests = [
+        "test_read_format_rar_set_format",
+        "test_read_format_rar_basic",
+        "test_read_format_rar_subblock",
+        "test_read_format_rar_noeof",
+        "test_read_format_rar_unicode_UTF8",
+        "test_read_format_rar_unicode_CP932",
+        "test_read_format_rar_compress_normal",
+        "test_read_format_rar_multi_lzss_blocks",
+        "test_read_format_rar_compress_best",
+        "test_read_format_rar_ppmd_lzss_conversion",
+        "test_read_format_rar_binary",
+        "test_read_format_rar_windows",
+        "test_read_format_rar_multivolume",
+        "test_read_format_rar_multivolume_w",
+        "test_read_format_rar_multivolume_skip",
+        "test_read_format_rar_sfx",
+        "test_read_format_rar_multivolume_stored_file",
+        "test_read_format_rar_multivolume_stored_file_skip",
+        "test_read_format_rar_multivolume_seek_data",
+        "test_read_format_rar_multivolume_seek_multiple_files",
+        "test_read_format_rar_multivolume_uncompressed_files",
+        "test_read_format_rar_ppmd_use_after_free",
+        "test_read_format_rar_ppmd_use_after_free2",
+    ],
+)
+libarchive_test(
+    name = "test_read_format_rar5",
+    deps = [":lib"],
+    tags = ["bcr_presubmit_skip"],  # TODO: failing on Windows
```

Comment:
Maybe just tag it incompatible on Windows so it runs on others?

Reply:
Hmm, this wasn't me, but happy do that.

Reply:
that was me initially - these are only test targets and are not public facing so I wouldn't be too worried about it. If you'd like to do it then great, but I wouldn't require the change myself.

Reply:
or can i leave this to somebody else as i need to fix zstd as well.